backend/controllers/cellMove/cellMovePrisonerSearch.js

```diff
@@ -0,0 +1,65 @@
+const { alertFlagLabels, cellMoveAlertCodes } = require('../../shared/alertFlagValues')
+const { putLastNameFirst, formatLocation } = require('../../utils')
+
+module.exports = ({ prisonApi }) => async (req, res) => {
+  const {
+    user: { activeCaseLoad },
+  } = res.locals
+  const { keywords } = req.query
+
+  if (!keywords) {
+    const errors =
+      keywords === undefined
+        ? undefined
+        : [
+            {
+              href: '#keywords',
+              html: 'Enter a prisoner&#8217;s name or number',
+            },
+          ]
+    return res.render('cellMove/cellMovePrisonerSearch.njk', {
+      hasSearched: false,
+      errors,
+    })
+  }
+
+  const currentUserCaseLoad = activeCaseLoad && activeCaseLoad.caseLoadId
+
+  const context = {
+    ...res.locals,
+    requestHeaders: {
+      'Sort-Fields': 'lastName,firstName',
+      'Sort-Order': 'ASC',
+    },
+  }
+
+  const [prisoners] = await Promise.all([
```

Comment:
We only use Promise.all when making multiple requests to APIs, as it makes them concurrently. Really all you need here is: `const prisoners = await  prisonApi.getInmates(context, currentUserCaseLoad, {
      keywords,
      returnAlerts: 'true',
    })`

Reply:
Ah yeah